tracing-attributes/src/attr.rs

```diff
@@ -268,6 +268,48 @@ impl Default for FormatMode {
     }
 }
 
+impl FormatMode {
+    /// Returns a list of the valid options for the format mode, this should be updated
+    /// when new modes are added to ensure that parser error messages are formatted correctly.
+    fn options() -> &'static [&'static str] {
+        &["Debug", "Display", "Value"]
+    }
```

Comment:
nit, take it or leave it: I feel like this may as well be a constant, rather than a function?